src/library_openal.js

```diff
@@ -1917,6 +1918,10 @@ var LibraryOpenAL = {
     c.capturedFrameCount = 0;
     c.isCapturing = false;
 
```

Comment:
a comment here explaining why this is needed would help, I think.

Reply:
resolved